### ISSUE TITLE
[RT 5.1] Table. Sticky bottom - зафиксированная нижняя строка имеет ховер

### ISSUE THREAD

**Issue by alexhawkins94** [zyfra/Prizm#2111]
Table
 Sticky bottom - зафиксированная нижняя строка имеет ховер. 
Вроде такого быть не должно и она должна быть аналогично строке заголовков?

![image](https://github.com/user-attachments/assets/e3b97831-bdfa-4e00-805a-e4c56fd12d3c)

_Originally posted by @alexhawkins94 in https://github.com/zyfra/Prizm/issues/2088#issuecomment-2399617309_

**Comment by PrizmDS** [comment]
Да, зафиксированная строка, как и статусная строка в таблице не должна иметь ховер.
